docs/api/instruments/andeenhagerling/ah2700a.rst

```diff
@@ -0,0 +1,7 @@
+###########################################
+Andeen Hagerling AH2700A capacitance bridge
+###########################################
+
+.. autoclass:: pymeasure.instruments.andeenhagerling.AH2700A
+    :members:
+    :show-inheritance:
```

Comment:
This should be fixed in c1cf9e0 by adding `":inherited-members: Instrument"` which adds members of `AH2500A` but not the ones of Instrument. From `Instrument` it might be interesting to add only some selected ones like `id`. I see that in some devices you show all inherited members and then exclude a list of those which one does not want. not sure how you propose to handle this here?

Reply:
I think I would do it depening on which number is smaller: the of the elements to exclude or the of the elements to include.

Reply:
with the latest changes (of "includeSCPI=False") this seems to be solved without further changes. things from AH2500A are shown and this should cover all necessary things.